eng/build.yml

```diff
@@ -14,201 +14,206 @@ parameters:
   # Job timeout
   timeoutInMinutes: 180
 
-jobs:
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - template: /eng/common/templates/job/onelocbuild.yml
+
+stages:
```

Comment:
I took a look at the code and is happening the same as the dotnet/installer repo. This is already declared on vsts-ci.yml
https://github.com/dotnet/sdk/blob/380758911b39a3e2cae2c1ab9f6a13dc539eef44/.vsts-ci.yml#L37
so you don't need to do it again (right now there is a conflict because there is a declaration of stages inside of a jobs declaration).

you can skip doing all this change and just doing the following change on vsts-ci.yml
```
stages:
- stage: build
   jobs:
      - job: Publish_Build_Configuration
        steps:
          - publish: $(Build.SourcesDirectory)\eng\BuildConfiguration
            artifact: BuildConfiguration
            displayName: Publish Build Config
      - template: /eng/build.yml
```